source/AI.cpp

```diff
@@ -1773,16 +1775,17 @@ void AI::DoScatter(Ship &ship, Command &command)
 
 
 
-Point AI::StoppingPoint(const Ship &ship, bool &shouldReverse)
+// Instead of coming to a full stop, adjust to a target velocity vector
+Point AI::AdjustPoint(const Ship &ship, const Point &targetVelocity, bool &shouldReverse)
```

Comment:
I can switch back to StoppingPoint() if you want. Originally I was working with 2 functions, as you can see from my commits, and then did a code cleanup afterwards by moving all calls of StoppingPoint to the new version with the targetVelocity included.
If you want me to change it back to AdjustPoint, I will happily do so tomorrow.
And then I will fix that missing 3rd line. I really thought I had 3 lines, since I was aware of that coding standard.